src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/ConfigTests.cs

```diff
@@ -788,6 +788,23 @@ public async Task DefaultFunctions()
             Assert.True(result.IsSuccess);
         }
 
+        // Verify we can mutate the config. 
+        [Fact]
+        public async Task DefaultFunctions2()
+        {
+            var config = new PowerFxConfig();
+            config.AddFunction(new Func1Function());
+            
+            var engine = new Engine(config);
+           
+            var result = engine.Check("Func1(3)"); // 6
+            Assert.True(result.IsSuccess);
+
+            config.AddFunction(new MultiplyFunction());
```

Comment:
This line is key - ensuring that even if the config is mutated, the invalidation is still working.